tools/StatelessExecution/StatelessExecution.csproj

```diff
@@ -2,9 +2,6 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net9.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
```

Comment:
Not sure why nullable removed here

Reply:
and implicit usings also

Reply:
[This](https://github.com/NethermindEth/nethermind/blob/fix/statelessexec-tool/tools/Directory.Build.props#L6-L7) is why